libprecompiled/CRUDPrecompiled.cpp

```diff
@@ -77,3 +168,84 @@ bytes CRUDPrecompiled::call(ExecutiveContext::Ptr context, bytesConstRef param,
     }
     return out;
 }
+
+void CRUDPrecompiled::parseCondition(
+    const std::string& conditionStr, const Table::Ptr& table, Condition::Ptr& condition)
+{
+    Json::Reader reader;
+    Json::Value conditionJson;
+    if (!reader.parse(conditionStr, conditionJson))
+    {
+        PRECOMPILED_LOG(ERROR) << LOG_BADGE("CRUDPrecompiled")
+                               << LOG_DESC("condition json parse error")
+                               << LOG_KV("condition", conditionStr);
+    }
+    else
+    {
+        condition = table->newCondition();
+        auto members = conditionJson.getMemberNames();
+        Json::Value OPJson;
+        for (auto iter = members.begin(); iter != members.end(); iter++)
+        {
+            OPJson = conditionJson[*iter];
+            auto op = OPJson.getMemberNames();
+            for (auto it = op.begin(); it != op.end(); it++)
+            {
+                if (*it == "eq")
+                {
+                    condition->EQ(*iter, OPJson[*it].asString());
+                }
+                else if (*it == "ne")
+                {
+                    condition->NE(*iter, OPJson[*it].asString());
+                }
+                else if (*it == "gt")
+                {
+                    condition->GT(*iter, OPJson[*it].asString());
+                }
+                else if (*it == "ge")
+                {
+                    condition->GE(*iter, OPJson[*it].asString());
+                }
+                else if (*it == "lt")
+                {
+                    condition->LT(*iter, OPJson[*it].asString());
+                }
+                else if (*it == "limit")
+                {
+                    std::string offsetCount = OPJson[*it].asString();
+                    std::vector<std::string> offsetCountList;
+                    boost::split(offsetCountList, offsetCount, boost::is_any_of(","));
+                    size_t offset = boost::lexical_cast<size_t>(offsetCountList[0]);
+                    size_t count = boost::lexical_cast<size_t>(offsetCountList[1]);
+                    condition->limit(offset, count);
+                }
+                else
```

Comment:
use `else if` instead `else`, use `else` to handle unexpected input.

Reply:
fix it